### PR TITLE
Added billing.Order

### PR DESCRIPTION
**Pull request by yash-pal1**

### CODE REVIEW

File: jelastic/api/billing/__init__.py [hunk 1]
```diff
@@ -36,6 +36,20 @@ def Account(self) -> "_Account":
...
+    def Order(self) -> "_Order":
+        """
+        The methods of this service provide billing information about a user account (such as UID, balance, billing history,
+        quotas, etc.) and allow managing it.
+
+        >>> from jelastic import Jelastic
+        >>> jelastic = Jelastic('https://jca.xapp.cloudmydc.com', token='d6f4e314a5b5fefd164995169f28ae32d987704f')
+        >>> jelastic.billing.Order
+
+        Ref: https://docs.jelastic.com/api/#!/api/billing.Order
```
Comment (abhi1693) [inline jelastic/api/billing/__init__.py:49]: The ref links do not match

File: jelastic/api/billing/__init__.py [hunk 2]
```diff
@@ -703,3 +717,139 @@ def WithdrawAccounts(self, start_date: date, end_date: date):
...
+
+    def EnvsResourcesByAccount(
+        self,
+        start_date: date,
+        end_date: date,
+        uid: int,
+        checksum: str,
+    ):
+        """
+        :param start_date: START DATE
```
Comment (abhi1693) [inline jelastic/api/billing/__init__.py:797]: When writing these, please use more descriptive sentences

File: jelastic/api/billing/__init__.py [hunk 3]
```diff
@@ -703,3 +717,139 @@ def WithdrawAccounts(self, start_date: date, end_date: date):
...
+        :param end_date: END DATE
+        :param uid: UID
+        :param checksum: required but not used
+        :return:
```
Comment (abhi1693) [inline jelastic/api/billing/__init__.py:801]: Do not add empty returns

**Review by abhi1693 — CHANGES_REQUESTED** [review]

**Review by abhi1693 — APPROVED** [review]

File: jelastic/api/billing/__init__.py [hunk 4]
```diff
@@ -747,6 +761,138 @@ def WithdrawAccounts(self, start_date: date, end_date: date):
         )
 
 
+class _Order(Billing):
```
Comment (abhi1693) [inline jelastic/api/billing/__init__.py:764]: _endpoint2 is missing

**Review by abhi1693 — CHANGES_REQUESTED** [review]

File: jelastic/api/billing/__init__.py (hunk 4 above)
Comment (yash-pal1) [reply jelastic/api/billing/__init__.py:722]: resolved

**Review by abhi1693 — APPROVED** [review]

**Pull request merged**